release-notes/aws/v10.1.2.md

```diff
@@ -0,0 +1,22 @@
+# :zap: Giant Swarm Release 10.1.2 for AWS is now active for you! :zap:
+
+This release fixes a problem with the cluster role for chart-operator, which is responsible for installing and updating Managed Apps. The changes affect only Managed Apps and upgrading to this release will not cause cluster nodes to be rolled. **To prevent encountering problems when installing or updating an app, please upgrade to this release.**
+
+### chart-operator [v0.11.3](https://github.com/giantswarm/chart-operator/releases/tag/v0.11.3)
+- Adjusted RBAC permissions.
+
+### cluster-autoscaler v1.16.2 ([Giant Swarm app v1.1.3](https://github.com/giantswarm/cluster-autoscaler-app/releases/tag/v1.1.3))
+- Updated manifests for Kubernetes 1.16 compatibility.
```

Comment:
Kind of weird since the release is k8s 1.15 :grimacing: 
But I get why it is the case.

Reply:
Yes I didn't to create a new cluster-operator release just for this. We have enough of those already.

Moving apps to releases will help. We're on the case.